### DISCUSSION TITLE
Boot from USB

### DISCUSSION THREAD

**Discussion by Banana71** [probonopd/MiniDexed#286] [Q&A] [answered]
Hi,
I have banned the Raspberry Pi in a case and would like to boot my MiniDexed from USB.
I tried it on my Raspberry Pi 3 Model B+ with no success. I don't want to use a micro SD card extension.
Is there a solution or can BareMetal only boot from the micro SD card?

Peter

**Comment by probonopd** [comment, marked as answer]
Hello Peter, as far as I know https://github.com/rsta2/circle/ which MiniDexed is using does not support booting from USB yet.

**Reply by Banana71** [reply]
Okay, thank you
